pages/aboutstyle.css

```diff
@@ -0,0 +1,379 @@
+/* stylelint-disable csstree/validator */
```

Comment:
- [ ] Kindly ensure that your linters pass by showing the green check, avoid disabling linters, it would be better to remove this line 👍🏿